### ISSUE TITLE
Changed Files not detected correctly on Windows

### ISSUE THREAD

**Issue by MattCarr147** [chromaui/chromatic-cli#546]
Hi, 

There seems to be a bug in how changed files are detected when running in a Windows environment. I believe this is preventing me from getting the TurboSnap feature to work correctly. 

An example snippet of the broken behaviour is shown below

```
Found 1 changed files:
  .gitignore
package.json
```

(Expected)

```
Found 2 changed files:
  .gitignore
  package.json
```

We've tested the same command (on the same repo etc) in a Linux environment and changes are detected correctly. We've also modified the cli code locally as follows and managed to get it picking up the changes on Windows too (but it's a bit of a hacky fix!). 

```
// exportasync function getChangedFiles(baseCommit: string,headCommit ='')

return files.split(EOL).filter(Boolean);

// changed to...

returnfiles.replace("EOL","\n").split("\n").filter(Boolean);
```

From what we could tell, the EOL value is correct `\r\n` but `returnFiles` (git?) appears to contain `\n` only, hence the bug.

**Comment by MattCarr147** [comment]
Hi, 

There seems to be a bug in how changed files are detected when running in a Windows environment. I believe this is preventing me from getting the TurboSnap feature to work correctly. 

An example snippet of the broken behaviour is shown below

```
Found 1 changed files:
  .gitignore
package.json
```

We've tested the same command (on the same repo etc) in a Linux environment and changes are detected correctly. We've also modified the cli code locally as follows and managed to get it picking up the changes on Windows too (but it's a bit of a hacky fix!). 

```
// exportasync function getChangedFiles(baseCommit: string,headCommit ='')

return files.split(EOL).filter(Boolean);

// changed to...

returnfiles.replace("EOL","\n").split("\n").filter(Boolean);
```

From what we could tell, the EOL value is correct `\n\r` but `returnFiles` (git?) appears to contain `\n` only, hence the bug.

**Comment by yngvebn** [comment]
@MattCarr147 I had problems with this myself, but with this https://github.com/chromaui/chromatic-cli/pull/539 released in Chromatic cli 6.5.2 it seems to be working as it should.

Did you also remember to pass the --storybook-base-dir to chromatic if your repository root is different from the folder where chromatic/storybook is run from?

**Comment by MattCarr147** [comment]
Just to confirm that I'm using CLI v6.5.2. As @yngvebn mentioned, the latest version was necessary to fix an issue I too was encountering previously (_"No CSF globs detected"_ etc). I've tried to keep everything as close to the defaults as possible in our config, so as far as I'm aware there should be no reason for me to pass any other flags.

The issue I've described above is unrelated to #539 from what I can see, but I'd hope that it's a simpler one in terms of a fix as it looks to just be flawed string/array processing.
